src/client/process.ts

```diff
@@ -99,7 +99,10 @@ export class PowerShellProcess {
         })
 
         if (!this.config.integratedConsole.showOnStartup) {
-            this.consoleTerminal.hide()
+            // TODO: remove `as any` when coc.nvim updates past 0.0.72-471db1a4c4
+            const buf = vscode.workspace.nvim.createBuffer((this.consoleTerminal as any).bufnr);
```

Comment:
we may want to monitor the presence of the terminal buffer by registering autocommands here.